XamlStyler3.Package/StylerPackage.cs

```diff
@@ -83,8 +84,7 @@ protected override void Initialize()
 
         private bool IsFormatableDocument(Document document)
         {
-            bool isFormatableDocument;
-            isFormatableDocument = !document.ReadOnly && document.Language == "XAML";
+            var isFormatableDocument = !document.ReadOnly && document.Language == "XAML";
```

Comment:
I am getting a bunch of warnings about ensuring these are only call on main thread. Can resolve with `ThreadHelper.ThrowIfNotOnUIThread();` to be safe